features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -615,6 +616,17 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
         } else {
             bottomSheet.peekHeight = bottomSheetPeekHeight
         }
+
+        if (viewModel.isLocationEnabled.value == true &&
+            viewModel.filterMode.value != FilterMode.Online && filters.territory.isEmpty()){
+            bottomSheet.isDraggable = true
+            bottomSheet.state = BottomSheetBehavior.STATE_HALF_EXPANDED
+            bottomSheetWasExpanded = false
+        } else {
+            bottomSheet.isDraggable = false
+            bottomSheet.state = BottomSheetBehavior.STATE_EXPANDED
+            bottomSheetWasExpanded = true
+        }
```

Comment:
Unfortunately, this creates some problems. First of all, `resolveAppliedFilters` method is only for setting up the bottom panel that shows what filters are applied, it shouldn't contain any other logic (maybe we should rename this method if its purpose isn't clear).

Second, there are numerous other places in the SearchFragment that change `isDraggable` and `state` and those places will ignore `territory` and this snippet completely.

The way I think this should be set up is we should have a single method that checks all conditions for the bottom sheet draggable status (similar to the `shouldShowFiltersPanel` method). Then, all assignments like `bottomSheet.isDraggable =` should use this method instead of a direct assignment. Conditions that should be checked are:
- is location enabled
- is online filter mode
- screen state

Similar method can be added for the `bottomSheet.state`. There, apart from the above-mentioned conditions merchant type should also be checked, since the sheet is always expanded for Online merchants.

Changes in the territory can continue to be observed through `appliedFilters`, or you can add a livedata for territory and observe it directly.